src/plugins/saml-auth/components/WrapAuthItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import PropTypes from "prop-types"
+import SamlAuth from "./SamlAuth"
+
+export default class WrappedAuthItem extends React.Component {
```

Comment:
sorry i don't understand. this component is not imported anywhere else?

Reply:
but i see `import WrappedSamlAuthItem from "./components/WrapAuthItem"` is this what it is supposed to be?